graphbolt/src/csc_sampling_graph.cc

```diff
@@ -221,17 +231,52 @@ torch::Tensor Pick(
       picked_neighbors =
           torch::randint(offset, offset + num_neighbors, {fanout}, options);
     } else {
-      picked_neighbors = torch::randperm(num_neighbors, options) + offset;
-      picked_neighbors = picked_neighbors.slice(0, 0, fanout);
+      picked_neighbors = torch::randperm(num_neighbors, options);
+      picked_neighbors = picked_neighbors.slice(0, 0, fanout) + offset;
     }
   }
   return picked_neighbors;
 }
 
+inline torch::Tensor NonUniformPick(
```

Comment:
Consider also copy the doc string to here as well? Since you already wrote pretty good doc string in the header file.

Reply:
Sure, let me make a copy.